includes/class-wcs-att-cart.php

```diff
@@ -20,7 +20,29 @@
  */
 class WCS_ATT_Cart {
 
+	/**
+	 * Flag to ensure hooks can be added only once.
+	 * @var bool
+	 */
+	private static $added_hooks = false;
```

Comment:
Always something to learn. Was probably over-zealous here. Will revert that check. 👍